### ISSUE TITLE
Installation error

### ISSUE THREAD

**Issue by roalheva** [nvdaes/plover_spanish_mqd#2]
When I change the system to Spanish MQD the errors that I attach in the picture appear.

I currently use Plover 4.0.0.dev8 + 66.g685bd33 on a macbook pro

![Screen Shot 2021-04-20 at 12 39 46](https://user-images.githubusercontent.com/79938874/115501032-18344080-a290-11eb-8ded-91e32af373f5.png)

How can I solve it?

**Comment by roskoff** [comment]
@benoit-pierre it seems it was wrongly released.

As a local workaround, you can edit the firsts lines of `~/Library/Application\ Support/plover/spanish_mqd_double.py`.

Replace this:
```
import os
import sys
sys.path.append(os.path.dirname(__file__))
import spanish_mqd_single
del sys.path[-1]
```

with this:
```
import os
from plover.oslayer.config import CONFIG_DIR

os.chdir(CONFIG_DIR)
import spanish_mqd_single
```

The problem about invalid keymaps can be solved from Machine Configuration, you just need to pair key `W-` with action `N-` and key `-B` with action `-n`.

**Comment by nvdaes** [comment]
Thanks, @roalheva for reporting. I'll release a new version so you can update it and check if the issue is fixed.
Kind regards

**Comment by nvdaes** [comment]
@roalheva, I've released version 2.0.1 of the plugin. Please install this version, where this issue should be fixed. I'll close this. Please feel free to reopen it or provide more feedback if you still find errors.
Thanks

**Comment by nvdaes** [comment]
Fixed in PR #3

**Comment by roskoff** [comment]
@nvdaes please review #4 too. The other PR (#3) fixed only one of the problems reported!
